python/cudf/cudf/io/parquet.py

```diff
@@ -41,10 +42,10 @@ def read_parquet(
         filepath_or_buffer, None, **kwargs
     )
     if compression is not None:
-        ValueError("URL content-encoding decompression is not supported")
+        raise ValueError("URL content-encoding decompression is not supported")
 
     if engine == "cudf":
-        df = libparquet.read_parquet(
+        return libparquet.read_parquet(
```

Comment:
Doesn't this return a Table now where you'd need to convert it to a DataFrame?

Reply:
So it did, I moved the Dataframe creation back to the pyx file because I needed it there to set the index names. I guess I could do parse that information in the .py file? What do you think? Want me to move it back to the .py file?

Reply:
I remember why I moved it back to the .pyx file now. I needed access directly to the table_with_metadata object to get the user_data where the Parquet json data is kept to get the information about the index